src/textbooks/textbook-sidebar/TextbookSidebar.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+
+import { Hyperlink } from '@edx/paragon';
+import { HelpSidebar } from '../../generic/help-sidebar';
+import messages from './messages';
+import { useHelpUrls } from '../../help-urls/hooks';
+
+const TextbookSidebar = ({ courseId }) => {
+  const intl = useIntl();
+  const { textbooks: textbookUrl } = useHelpUrls(['textbooks']);
+
+  return (
+    <HelpSidebar courseId={courseId} className="pt-4">
+      <h4 className="help-sidebar-about-title">
+        {intl.formatMessage(messages.section_1_title)}
+      </h4>
+      <p className="help-sidebar-about-descriptions">
+        {intl.formatMessage(messages.section_1_descriptions)}
+      </p>
+      <hr className="my-3.5" />
+      <h4 className="help-sidebar-about-title">
+        {intl.formatMessage(messages.section_2_title)}
+      </h4>
+      <p className="help-sidebar-about-descriptions">
+        {intl.formatMessage(messages.section_2_descriptions)}
+      </p>
+      <Hyperlink
+        className="small"
+        destination={textbookUrl}
+        target="_blank"
+        showLaunchIcon={false}
+      >
+        {intl.formatMessage(messages.section_link)}
```

Comment:
```suggestion
        {intl.formatMessage(messages.sectionLink)}
```
If we don't have number in the message name then we use `camelCase`.
Other examples that I found in the project. I think we will follow this pattern:

```
messages.section_3_link
messages.section_4_descriptions_1
messages.alertSuccessAriaLabelledby
```

I see there is example in the project that I don't like `messages.defaultSection_1_Title`. Ideally it should be `messages.defaultSection_1_title`

Reply:
Fixed